helm-charts/seldon-core-operator/values.yaml

```diff
@@ -89,6 +89,8 @@ manager:
   containerSecurityContext: {}
   deploymentNameAsPrefix: false
   priorityClassName:
+  nodeSelector: {}
```

Comment:
This should actually be a list as opposed to a dictionary (https://kubernetes.io/docs/tasks/configure-pod-container/assign-pods-nodes/)